content/this-month/2023-06/index.md

```diff
@@ -50,6 +50,23 @@ Please follow this template:
 
 In this section, we give an overview of notable changes to the projects hosted under the [`rust-osdev`] organization.
 
+### [`multiboot2`](https://github.com/rust-osdev/multiboot2)
+<span class="maintainers">Maintained by [@phip1611](https://github.com/phip1611)</span>
+
+The [multiboot2](https://crates.io/crates/multiboot2) and the [multiboot2-header](https://crates.io/crates/multiboot2)
+crates got a big overhaul. The list of new features includes but is not limited to:
+- use DSTs for tags where applicable ([#134](https://github.com/rust-osdev/multiboot2/pull/134))
+- model the MBI as DST ([#155](https://github.com/rust-osdev/multiboot2/pull/155))
+- add a runtime builder for an MBI ([#133](https://github.com/rust-osdev/multiboot2/pull/133))
+  Huge thanks to [YtvwlD / Niklas](https://github.com/YtvwlD) for this greate external contribution
```

Comment:
Nit: Markdown currently renders this on the same line. To start a new paragraph you need two newlines. Alternatively, we could separate it by a comma.